style.css

```diff
@@ -0,0 +1,71 @@
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-size: 20px;
+  color: white;
+  background-color: purple;
+  font-family: 'Trebuchet MS', sans-serif;
+}
+
+h1{
+  line-height: 2.4;
+  font-weight: 500;
+  font-size: 48px;
+ 
```

Comment:
Usually, it is best to not have selectors have extra empty lines, if you intend to use it in the future for future styling, then it would be best to add said lines until then. 

There are a couple of these throughout the document, please make this change.